keps/sig-network/20190613-kube-proxy-component-config.md

```diff
@@ -0,0 +1,210 @@
+---
+title: Kube-Proxy ComponentConfig graduation
+authors:
+  - "@rosti"
+owning-sig: sig-network
+participating-sigs:
+  - sig-cluster-lifecycle
+  - sig-api-machinery
+  - wg-component-standard
+reviewers:
+  - "@luxas"
+  - "@mtaufen"
+  - "@sttts"
+  - TBD
+approvers:
+  - "@thockin"
+  - TBD
+editor: "@rosti"
+creation-date: 2019-06-13
+status: implementable
+---
+
+# kube-proxy component config graduation proposal
+
+## Table of Contents
+
+* [kube-proxy component config graduation proposal](#kube-proxy-component-config-graduation-proposal)
+  * [Table of Contents](#table-of-contents)
+  * [Release Signoff Checklist](#release-signoff-checklist)
+  * [Summary](#summary)
+  * [Motivation](#motivation)
+    * [Goals](#goals)
+    * [Non-Goals](#non-goals)
+  * [Proposal](#proposal)
+    * [Re-encapsulate mode specific options](#re-encapsulate-mode-specific-options)
+      * [Example](#example)
+    * [Group platform specific fields together](#group-platform-specific-fields-together)
+      * [Example](#example-1)
+    * [Split instance specific and shared settings](#split-instance-specific-and-shared-settings)
+      * [Example](#example-2)
+    * [Risks and Mitigations](#risks-and-mitigations)
+  * [Design Details](#design-details)
+    * [Test Plan](#test-plan)
+    * [Graduation Criteria](#graduation-criteria)
+
+## Release Signoff Checklist
+
+**ACTION REQUIRED:** In order to merge code into a release, there must be an issue in [kubernetes/enhancements] referencing this KEP and targeting a release milestone **before [Enhancement Freeze](https://github.com/kubernetes/sig-release/tree/master/releases)
```

Comment:
@rosti, shall we tag #784 /w 1.16 and update the KEP?

Reply:
@stealthybox 
@rosti is on PTO, but i will defer to him on comments for the milestone.
let me know if you need help updating #784.